renv.lock

```diff
@@ -0,0 +1,1811 @@
+{
```

Comment:
Is this file intended to be in the PR?

Reply:
It was not! Thanks for catching both of those.